templates/resources/dw_vw_impala.md.tpl

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
The reason why the rendering is not correct is that this file name is templates/resources/dw_vw_impala.md.**tpl** and it should be templates/resources/dw_vw_impala.md.**tmpl**

Once I render with the good file name this works nicely